webapp/indexSelfContained.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>GeometryApp</title>
+	<script id="sap-ui-bootstrap"
+		src="resources/sap-ui-custom.js"
+		data-sap-ui-libs="sap.m"
+		data-sap-ui-theme="sap_belize"
+		data-sap-ui-compatVersion="edge"
+		xx-data-sap-ui-preloadLibCss="!sap.ui.core,sap.m"
```

Comment:
this line does not seem to work - at least for me, library CSS is still loaded for all 3 libs individually

Reply:
That's because it starts with `xx-` which so to say invalidates the configuration. This seems to come from https://github.com/SAP/openui5-sample-app/blob/07f901c3146782d4cfad4b4ab3837df0405c8141/webapp/index-self-contained.html#L17 which also as a link tag commented out as bundling of CSS is not part of the UI5 Tooling, yet see https://github.com/SAP/ui5-builder/issues/91

Reply:
alright, let's revert it for now then

Reply:
I would opt for removing them, as it's confusing and doesn't add value. Once this feature is available, the file anyway needs to be adopted.